Dockerfile

```diff
@@ -5,6 +5,22 @@ MAINTAINER Juan Marin Otero <juan.marin.otero@gmail.com>
 RUN yum -y install epel-release
 RUN yum -y install nginx
 
+# Install npm
+RUN yum install -y npm
+
+# Install grunt-cli
+RUN npm install -g grunt-cli
+
+COPY . /
+
+RUN npm install
+
+# Get node-sass to work
+RUN node ./node_modules/grunt-sass/node_modules/node-sass/scripts/install.js;
+
+RUN grunt build
+
+# grunt builds to dist
 ADD dist /opt/grasshopper-ui/
```

Comment:
Since you're doing a `COPY` above and then building, you probably don't want to then add the local `dist` directory to `/opt/grasshopper-ui`.  Seems like you could just remove this line, and then change `nginx.conf` to point to the built `dist` directory...which I think would be `/usr/src/app/dist`.